cul.js

```diff
@@ -96,11 +138,11 @@ function checkConnection(host, port, timeout, callback) {
 }
 
 function checkPort(callback) {
-    if (adapter.config.type === 'cuno') {
+    if(adapter.config.type === 'cuno') {
```

Comment:
Code was correct here and also regardng the semicolon below according to the "commoon codeing guidelines" ... but I would accept it

Reply:
done